Program.cs

```diff
@@ -0,0 +1,405 @@
+﻿using System;
+using System.Linq;
+using System.Collections.Generic;
+using VisualCode.Entities;
+
+namespace VisualCode
+{
+    class Program
+    {
+        enum enumMenuMain : int {Maintenance = 1, RegistrySales = 2, Reports = 3};
+        enum enumMenuMaintenance : int {Customers = 1, Products = 2, Sellers = 3, MenuPrincipal = 4};
+        enum enumMenuReport : int {Sales = 1, CustomersAge = 2, CustomersBirthdayMonth = 3, ProductsStock = 4};
+
+        static List<Customers> customersList = new List<Customers>();
+        static List<Products> productsList = new List<Products>();
+        static List<Sellers> sellersList = new List<Sellers>();
+        static List<Invoices> invoicesList = new List<Invoices>();
+        static List<InvoiceDetails> invoiceDetailsList = new List<InvoiceDetails>();
+
+        static void Main(string[] args)
+        {
+            LoadDataDefault();
+            MenuMain();
+            ProcessMain();
+        }
+        private static void ProcessMain()

```

Comment:
between methods must to be a blank line to separate and make the code more easy to read and understand.